packages/appsync-modelgen-plugin/src/visitors/appsync-model-introspection-visitor.ts

```diff
@@ -66,11 +67,17 @@ export class AppSyncModelIntrospectionVisitor<
       // Skip the field if the field type is union/interface
```

Comment:
This comment block needs update based on the updated check on line 70.

Reply:
Makes sense. I added that clarification in a separate line to make sure it's not confused with the TODO comment. Addressed in [9205ea4](https://github.com/aws-amplify/amplify-codegen/pull/869/commits/9205ea4c9412ec5907662c20fc36c4fdc7aae6a6)